boards/arm/efm32pg_stk3402a/efm32pg_stk3402a-testplan.json

```diff
@@ -0,0 +1,1278 @@
+{
```

Comment:
Why is this file being added?

Reply:
To only run tests that I have proven to pass.